packages/image_picker/image_picker/ios/Classes/FLTPHPickerImageHandlingOperation.m

```diff
@@ -0,0 +1,132 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FLTPHPickerImageHandlingOperation.h"
+
+API_AVAILABLE(ios(14))
+@interface FLTPHPickerImageHandlingOperation ()
+
+@property(strong, nonatomic) PHPickerResult *result;
+@property(weak, nonatomic) NSMutableArray *pathList;
+@property(assign, nonatomic) NSNumber *maxHeight;
+@property(assign, nonatomic) NSNumber *maxWidth;
+@property(assign, nonatomic) NSNumber *desiredImageQuality;
+@property(assign, nonatomic) NSInteger index;
+
+@end
+
+@implementation FLTPHPickerImageHandlingOperation {
```

Comment:
Can you think of a more descriptive class name?
Maybe something like `GetPathForPHPPickerImageOperation`?

Reply:
Maybe: `FTLPHPickerResultPopulatingOperation?`
Also comment on the class to say what this operation do, particularly we need to mention that `pathList` will get populated with the final results.